src/components/FilterPanel/FilterSearch/FilterSearch.js

```diff
@@ -1,188 +1,58 @@
 /**
- * @file Filter search.
- * @copyright IBM Security 2019
+ * @file Filter search component.
+ * @copyright IBM Security 2020
  */
 
-import classnames from 'classnames';
+import React from 'react';
 import PropTypes from 'prop-types';
-import React, { Component } from 'react';
-
-import { getComponentNamespace } from '../../../globals/namespace';
-import * as defaultLabels from '../../../globals/nls';
-
-import FilterSelector from '../FilterSelector';
-import { filterDataPropTypes, getFiltersArray } from '../FilterPanelUtilities';
+import classnames from 'classnames';
 import Search from '../../Search';
+import { getComponentNamespace } from '../../../globals/namespace';
 
-export const namespace = getComponentNamespace('filter-search');
-
-class FilterSearch extends Component {
-  state = {
-    searchValue: this.props.initialSearchValue,
-  };
-
-  /**
-   * Handles filter toggling.
-   * @param {Filter} filter Filter object to toggle.
-   */
-  onFilterSelection = filter => {
-    this.setState({ searchValue: '' });
-    this.props.onChange(filter);
-  };
+const namespace = getComponentNamespace('filter-search');
+
+const FilterSearch = ({ children, className, ...other }) => {
+  const [displayResults, setDisplayResults] = React.useState(false);
+
+  const hideResultsIfLostFocus = ({ currentTarget }) =>
+    setTimeout(() => {
+      if (!currentTarget.contains(document.activeElement)) {
+        setDisplayResults(false);
+      }
+    }, 300);
+
+  return (
+    <div
+      className={namespace}
+      onBlur={hideResultsIfLostFocus}
+      onFocus={() => setDisplayResults(true)}
+    >
+      <Search
+        size="sm"
+        className={classnames(`${namespace}__input`, className)}
+        {...other}
```

Comment:
Could you put the spread on the parent/root element?

Reply:
Same thinking about exposing the `Search` component to consumers.

Reply:
Yeah that makes sense 👍